packages/protocol/contracts/stability/StableToken.sol

```diff
@@ -127,6 +130,67 @@ contract StableToken is
     setRegistry(registryAddress);
   }
 
+  /**
+   * @notice Setter for the total supply.
+   * @param totalSupply The total supply to set.
+   */
+  function setTotalSupply(uint256 totalSupply) public {
+    totalSupply_ = totalSupply;
+    emit TotalSupplySet(msg.sender, totalSupply);
+  }
+
+  /**
+   * @notice Setter for the name of the token.
+   * @param name The name to set.
+   */
+  function setName(string memory name) public {
+    name_ = name;
+    emit TokenNameSet(msg.sender, name);
+  }
+
+  /**
+   * @notice Setter for the symbol of the token.
+   * @param symbol The symbol to set.
+   */
+  function setSymbol(string memory symbol) public {
+    symbol_ = symbol;
+    emit TokenSymbolSet(msg.sender, symbol);
+  }
+
+  /**
+   * @notice Setter for the decimals of the token.
+   * @param decimals The decimals to set.
+   */
+  function setDecimals(uint8 decimals) public {
+    decimals_ = decimals;
+    emit TokenDecimalslSet(msg.sender, decimals);
+  }
+
+  /**
+   * @notice Sets initial inflation Parameters.
+   * @param inflationRate new rate.
```

Comment:
Comment that unwrapped Fixidity value expected for `inflationRate`.

Reply:
done